conf/app.routes

```diff
@@ -96,6 +96,9 @@ GET         /update-information/declaration
 
 GET         /update-information/add-person                                            @controllers.declaration.AddPersonController.getWithAmendment()
 
+GET         /decleration/renew-registration                                           @controllers.declaration.RenewRegistrationController.get()
```

Comment:
Spelling mistake in declaration - URL will be wrong. Might affect acceptance tests if they've been done inline with this.